packages/bpk-component-datatable/src/BpkDataTable.js

```diff
@@ -177,14 +116,15 @@ class BpkDataTable extends Component {
         {...restOfProps}
         className={classNames.join(' ')}
         width={width}
-        rowCount={sortedList.length}
-        rowGetter={({ index }) => sortedList[index]}
+        rowCount={this.state.sorter.rowCount}
+        rowGetter={({ index }) => this.state.sorter.getRow(index)}
         headerClassName={headerClassNames.join(' ')}
         rowClassName={row => this.rowClassName(rowClassName, row)}
         onRowClick={this.onRowClicked}
         onHeaderClick={this.onHeaderClick}
-        sortBy={sortBy}
-        sortDirection={sortDirection}
+        sortBy={this.state.sorter.sortProps.sortBy}
```

Comment:
This part could be something like

```javascript
{...this.state.sorter.sortProps()}
```

and we could move the repsonsibility of providing `sort`, `sortDirection`, and `sortBy` to the sorter